scripts/prepare_datasets/prepare_datasets.py

```diff
@@ -77,8 +74,30 @@
     lines.append(f'| {sortid} | [view]({le_url}) | {description} |')
 lines.append('')
 
+lines.append('')
+lines.append(f'[Browse all recordings]({aws_url}/default?feed={x.get_uri()})')
+
 txt = '\n'.join(lines)
 print('~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~')
 print(txt)
 print('~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~')
 
+try:
+    with open('README.md') as f:
+        readme_txt = f.read()
+    ind1 = readme_txt.find('<!-- BEGIN DATA TABLE -->')
+    ind2 = readme_txt.find('<!-- END DATA TABLE -->')
+    assert (ind1 >=0) and (ind2 >= 0)
+    readme_txt = ''.join([
+        readme_txt[:ind1],
+        '<!-- BEGIN DATA TABLE -->\n',
+        f'\n<!--- Auto-generated at {datetime.datetime.now().strftime("%m/%d/%Y, %H:%M:%S")}-->\n',
+        '\n'.join(lines),
+        '\n',
+        readme_txt[ind2:]
```

Comment:
For my sanity's sake, the way the index slicing works is this will overwrite the existing BEGIN_DATA_TABLE line but the next one will *not* overwrite the existing END_DATA_TABLE line, is that correct?

Reply:
yes. maybe that should be made more clear.